### PR TITLE
New stepper outcome

### PR DESCRIPTION
**Pull request by michaelvoronov**

### CODE REVIEW

**Review by folex — APPROVED** [review]

**Review by DieMyst — APPROVED** [review]

File: crates/stepper-interface/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "stepper-interface"
```
Comment (folex) [inline crates/stepper-interface/Cargo.toml:2]: Maybe add a description?

File: stepper-lib/src/errors.rs [hunk 2]
```diff
@@ -93,6 +91,34 @@ pub enum AquamarineError {
     ShadowingError(String),
 }
 
+impl AquamarineError {
```
Comment (folex) [inline stepper-lib/src/errors.rs:94]: I'm so happy this mapping finally found its true place!

File: stepper/src/fce.rs [hunk 3]
```diff
@@ -43,7 +43,7 @@ pub fn main() {
 }
 
 #[fce]
-pub fn invoke(init_peer_id: String, aqua: String, prev_data: String, data: String) -> StepperOutcome {
+pub fn invoke(init_peer_id: String, aqua: String, prev_data: Vec<u8>, data: Vec<u8>) -> StepperOutcome {
```
Comment (folex) [inline stepper/src/fce.rs:46]: Maybe change major version of fce, fluence-faas and aquamarine-vm due to this change?

**Review by folex — APPROVED** [review]

**Review by folex — APPROVED** [review]

File: stepper-lib/src/errors.rs (hunk 2 above)
Comment (michaelvoronov) [reply stepper-lib/src/errors.rs:94]: And me too :)

File: stepper/src/fce.rs (hunk 3 above)
Comment (michaelvoronov) [reply stepper/src/fce.rs:46]: Done

File: crates/stepper-interface/Cargo.toml (hunk 1 above)
Comment (michaelvoronov) [reply crates/stepper-interface/Cargo.toml:2]: Done

**Pull request merged**